app/exporters/formatters/manual_indexable_formatter.rb

```diff
@@ -7,20 +7,10 @@ def type
 
 private
   def extra_attributes
```

Comment:
not really. maybe? could drop [this](https://github.com/alphagov/specialist-publisher/blob/c7551e2b8525b1a088253982034a180683c250af/app/exporters/formatters/abstract_indexable_formatter.rb#L32) but I'd rather keep changes in this app minimal.